radio/apps/schedules/models.py

```diff
@@ -61,11 +62,11 @@ def clean(self):
                 if self.start_date > self.end_date:
                     raise ValidationError(_('end date must be greater than or equal to start date.'))
                 # check date collision
-                qs = ScheduleBoard.objects.filter(
-                    start_date__lte=self.end_date, end_date__isnull=True
-                ) | ScheduleBoard.objects.filter(
-                    start_date__lte=self.end_date, end_date__gte=self.start_date
+                qs = (
+                    ScheduleBoard.objects.filter(start_date__lte=self.end_date, end_date__isnull=True) |
+                    ScheduleBoard.objects.filter(start_date__lte=self.end_date, end_date__gte=self.start_date)
```

Comment:
Make code more readable. The important part is to apply multiple filters, so the code should not spread this across multiple indentation levels, but should reflect this by intuitive reading.